Symfony/CS/ConfigInterface.php

```diff
@@ -59,6 +59,8 @@ public function getFixers();
      * Sets the root directory of the project.
      *
      * @param string $dir The project root directory
+     *
+     * @return Config The same instance
```

Comment:
Nope, `@return $this` won't help on typehinting in IDE.
It should be `ConfigInterface` here.